### ISSUE TITLE
Error occurred while enabling

### ISSUE THREAD

**Issue by Qinak** [GeorgH93/MarriageMaster#57]
Doesn't get enabled.

**Spigot Version:** CraftBukkit version git-Spigot-1a3504a-dfa7583 (MC: 1.13.2)
That is the final spigot build for 1.13.2.
**MM Version:** 1.31.16

https://pastebin.com/gRZrQ1qF

**Comment by GeorgH93** [comment]
There is a problem with your config file. The `PrivateFormat` option is either missing or not a valid string.

**Comment by Qinak** [comment]
Thank you, it works.

Now I made a mistake in this thread. I forgot I was going to use V2 for the json text. My apologies. Downloading the latest MM V2 and PluginLib from ci.pcgamingfreaks.at gives me this on startup:
https://pastebin.com/qxntZqTX

I have removed the old MarriageMaster and PCGF_PluginLib folder from the plugins folder trying to generate new ones and I've also tried to only have MM and PluginLib in the plugins folder but I get the same error.
